util/cmdgen.py

```diff
@@ -89,7 +123,7 @@ def cmdgen_rewrite_md(filepath: Path, dry_run: bool, update: bool) -> bool:
             continue
 
         # Run the found-command in a subshell
-        res = subprocess.run(cmd, shell=True, capture_output=True)
+        res = subprocess.run(" ".join(cmd_args.cmd), shell=True, capture_output=True)
```

Comment:
```suggestion
        res = subprocess.run(" ".join(cmd_args.cmd), shell=True, capture_output=True , env={"COLUMNS": "80"})
```

Makes slightly more deterministic